wingpanel-interface/BackgroundManager.vala

```diff
@@ -105,31 +108,31 @@ public class WingpanelInterface.BackgroundManager : Object {
         current_workspace.window_added.connect (on_window_added);
         current_workspace.window_removed.connect (on_window_removed);
 
-        check_for_state_change (AnimationSettings.get_default ().workspace_switch_duration);
+        check_for_state_change (WORKSPACE_SWITCH_DURATION);
     }
 
     private void register_window (Meta.Window window) {
         window.notify["maximized-vertically"].connect (() => {
-            check_for_state_change (AnimationSettings.get_default ().snap_duration);
+            check_for_state_change (SNAP_DURATION);
         });
 
         window.notify["minimized"].connect (() => {
-            check_for_state_change (AnimationSettings.get_default ().minimize_duration);
+            check_for_state_change (MINIMIZE_DURATION);
         });
 
         window.workspace_changed.connect (() => {
-            check_for_state_change (AnimationSettings.get_default ().minimize_duration);
+            check_for_state_change (MINIMIZE_DURATION);
```

Comment:
This is correctly based on master, but is it right that the `workspace_changed` signal closure uses `MINIMIZE_DURATION" and not`"WORKPLACE_SWITCH_DURATION`?   This can be left for another PR (if required).

Reply:
Oh wow. Thank you for actually reading this! I feel dumb for not noticing this :sweat_smile: